### DISCUSSION TITLE
filter by or condition

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7434] [Help] [answered]
I want to filter data from various conditions like title, status, and auto id  
how can I achieve this?

**Comment by andz-bb** [comment, marked as answer]
Hey @DevrayePatil, are you talking about filtering data on a table? If so, the easiest way to filter the data is by adding a Dynamic Filter component under the Data Provider that is supplying your table with data. You can set up multiple conditions for the filter in the front end. There's more info about it [on our docs page here.](https://docs.budibase.com/docs/searchfilter-data)

**Reply by melohagan** [reply]
There's also an issue open for 'OR' filtering here: https://github.com/Budibase/budibase/issues/2585
The feature has been implemented, and should be released pretty soon!
